node-problem-detector.yaml

```
@@ -30,18 +31,28 @@ pipeline:
     with:
       patches: fix-archs.patch
 
+  - uses: patch
```

```suggestion
  # This is needed to account for breaking changes in prometheus/client_golang
  - uses: patch
```

Correct?

This will help us know to remove the patch after we remove the `go get` bump.

---

Correct!